npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -217,7 +217,7 @@ function createFakeContext (projectRoot: string, defaultProjectConfig: Cypress.A
     getProjectMetadata: () => {
       return {
         root: defaultProjectConfig.root,
-        sourceRoot: defaultProjectConfig.root,
```

Comment:
Did this API change between majors? Should we check `defaultProjectConfig.root ?? defaultProjectConfig.sourceRoot`